crates/ruff_dev/src/generate_docs.rs

```diff
@@ -33,7 +34,26 @@ pub(crate) fn main(args: &Args) -> Result<()> {
 
             let (linter, _) = Linter::parse_code(&rule.noqa_code().to_string()).unwrap();
             if linter.url().is_some() {
-                output.push_str(&format!("Derived from the **{}** linter.", linter.name()));
+                let codes_dsv: String = match linter.common_prefix() {
```

Comment:
What's the meaning of `dsv`? Maybe consider using a different name

Reply:
Renamed to `common_prefix`.